### PR TITLE
Feat/feeds feeds feeds

### PR DESCRIPTION
**Pull request by gingeranyhow**

### CODE REVIEW

File: client/src/components/Feed.jsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ class Feed extends React.Component {
 
   render() {
 
-    let hasTransactions = this.props.transactions.items && this.props.transactions.items.length > 0;
-    let showMoreButton = Boolean(this.props.transactions.nextPageTransactionId);
+    let hasTransactions = this.props.transactions && this.props.transactions.items && this.props.transactions.items.length > 0;
```
Comment (acenine) [inline client/src/components/Feed.jsx:13]: Unclear why these changes were neccessary.

File: client/src/components/Profile.jsx [hunk 2]
```diff
@@ -4,18 +4,89 @@ import Payment from './Payment.jsx';
...
+
+    // If feed was empty, set the returned transactions as the feed
+    let isFeedEmpty = !this.state[feedType].count || this.state[feedType].count === 0;
+
+    let newFeedObject = isFeedEmpty
+      ? transactionSummary
+      : feedManipulation.mergeFeeds(transactionSummary, this.state[feedType]);
+
+    this.setState({
+      [feedType]: newFeedObject
```
Comment (acenine) [inline client/src/components/Profile.jsx:60]: Why the square brackets?

Comment (gingeranyhow) [reply client/src/components/Profile.jsx:60]: @acenine - it causes Javascript to evaluate the variable feedType, rather than setting a key at the string feedType.

File: client/src/components/Feed.jsx (hunk 1 above)
Comment (gingeranyhow) [reply client/src/components/Feed.jsx:13]: They weren't - I thought they made the code clearer.

Comment (gingeranyhow) [reply client/src/components/Feed.jsx:13]: Also - they protect the app in case no data is passed. if there are no transactions, I don't want the string "this.props.transactions.nextPageTransactionId" to even attempt to evaluate.

File: client/src/index.jsx [hunk 3]
```diff
@@ -67,98 +68,59 @@ class App extends React.Component {
...
-      .catch((err) => {
-        console.error(err);
-      });
-  }
+  getFeed(feedType, userId = null, sinceId) {
+    let endpoint = feedManipulation.returnFeedEndpoint(feedType, userId);
 
-  getGlobalFeed(sinceId = null) {
-    let additionalData = {params: {sinceId: sinceId}}
+    let params = {
```
Comment (acenine) [inline client/src/index.jsx]: could do 'let {params} = sinceId;'
